engine/src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -502,6 +502,61 @@ class CromwellApiServiceSpec extends AsyncFlatSpec with ScalatestRouteTest with
         }
     }
 
+    behavior of "REST API /query GET endpoint"
+    it should "return labels if specified in query additionalKeys param" in {
+      Get(s"/workflows/$version/query?additionalKeys=labels&id=${CromwellApiServiceSpec.ExistingWorkflowId}") ~>
+        akkaHttpService.workflowRoutes ~>
+        check {
+          status should be(StatusCodes.OK)
+          contentType should be(ContentTypes.`application/json`)
+          val results = responseAs[JsObject].fields("results").convertTo[Seq[JsObject]]
+          results.head.fields("id") should be(JsString(CromwellApiServiceSpec.ExistingWorkflowId.toString))
+          results.head.fields(WorkflowMetadataKeys.Labels).asJsObject.fields("key1") should be(JsString("label1"))
+          results.head.fields(WorkflowMetadataKeys.Labels).asJsObject.fields("key2") should be(JsString("label2"))
+        }
+    }
+
+    behavior of "REST API /query GET endpoint"
```

Comment:
You should only need this line once, per `behavior of` name, then it's carried forwards by the `it`s